Server/Services/spotifyServices.js

```diff
@@ -0,0 +1,222 @@
+/**
+ * @constant SpotifyWebApi
+ * @module spotify-web-api-node
```

Comment:
Module name have to be "spotifyService". Then you can make documentation for constant spotifyWebAPI